py-polars/tests/unit/test_cwc.py

```diff
@@ -169,3 +168,29 @@ def test_issue_16436() -> None:
         .fill_nan(0)
         .collect()
     )
+
+
+def test_issue_16435() -> None:
```

Comment:
Can you give it a name of what you test (and then keep the number of the issue in the name).